core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -104,13 +126,10 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
         .map(n => new ProducerRecord(topic1, partition0, null: Array[Byte], n.toString))
```

Comment:
I'll look at this more closely after finishing the pull request.   The issue might be resolved after making the above changes to ```produceMessages```.  If not it sounds like a timing issue.